docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        file_obj = BytesIO()
</pre><p>Why declare this? Can you just write <code>df.to_csv(BytesIO())</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Done